src/firebase/firebase.js

```diff
@@ -0,0 +1,19 @@
+import * as firebase from 'firebase/app';
+import 'firebase/database';
+
+const firebaseConfig = {
```

Comment:
you could prepare some validating function, which will check if all required keys exists in `process.env`. If not, throw not caught error which will log in terminal lacking key.

Reply:
done, but totally not sure if it's even close to be ok